src/renderer/views/pools/PoolsOverview.style.tsx

```diff
@@ -0,0 +1,62 @@
+import * as A from 'antd'
+import styled from 'styled-components'
+import { palette } from 'styled-theme'
+
+import { Button as UIButton, ButtonProps as UIButtonProps } from '../../components/uielements/button'
+import { Label } from '../../components/uielements/label'
+import { media } from '../../helpers/styleHelper'
+
+export const Tabs = styled(A.Tabs)`
+  ${media.sm`
+    flex-grow: 1;
+
+    & .ant-tabs-content-holder {
+      display: flex;
+    }
+  `}
+`
+
+export const TabButtonsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  ${media.sm`
+    flex-direction: row;
+  `}
+  margin-bottom: 10px;
+`
+
+type TabButtonProps = UIButtonProps & { selected: boolean }
+
+// A wrapper around `UIButton` to provide custom props - needed for custom styles
+const TabButtonWrapper: React.FC<TabButtonProps> = ({ children, ...otherProps }) => (
+  <UIButton {...otherProps}>{children}</UIButton>
+)
+
+export const TabButton = styled(TabButtonWrapper).attrs<TabButtonProps>(({ selected }) => ({
+  typevalue: 'transparent',
+  weight: selected ? 'bold' : 'normal'
+}))`
+  font-size: 16px !important;
+  color: ${({ selected }) => palette('text', selected ? 1 : 0)} !important;
```

Comment:
actually no need to add additional wrapper component. there is `SC` "specific" syntax to  implement what you want:
```ts
export const TabButton = styled(UIButton).attrs<TabButtonProps>(({ selected }) => ({
  typevalue: 'transparent',
  weight: selected ? 'bold' : 'normal'
}))<TabButtonProps>` <==== this one
  ...
`
```